examples/Preprocessing_example.md

```diff
@@ -0,0 +1,77 @@
+# Preprocessign an Image
+
+The main objective of the Preprocessing phase is to make as easy as possible for the OCR system to distinguish a character/word from the background. There are several techniques used for Preprocessing but this **Preprocessor** class can do it without you going into details and jargon of traditional methods.
+
+`!pip install -U ocred --no-cache --force-reinstall`
+
+```
+# manually preprocessing an image
+import cv2
+from scipy import ndimage
+from ocred import Preprocessor
+from google.colab.patches import cv2_imshow
+
+
+preprocessed = Preprocessor("/content/images/CosmosOne.jpg")
+```
+
+```
+# scan the image and copy the scanned image
+scanned = preprocessed.scan(inplace=True)
+orig = scanned.copy()
+
+cv2_imshow(orig)
```

Comment:
It might be good to show users the `inplace=False` version and then point to the `inplace=True` version.

`inplace=True` edits the `overriden_image` that is passed to the the function, which is memory-wise efficient (hence time-wise efficient - the time taken by OS to create a new array for the copy of image is saved), but you might not want the program to alter your original image.

`inplace=False` on the other hand creates a copy of the passed image, hence more memory and time usage but does not alter the original image -
https://github.com/Saransh-cpp/OCRed/blob/420aa7a78e75e1ea7963079175260997a23816a4/ocred/preprocessing.py#L84-L89

If an `overriden_image` is not passed, then this is performed with the image passed to the class.

```suggestion
scanned = preprocessed.scan()

cv2_imshow(scanned)
```